src/vendor/wordpress/gutenberg.stories.mdx

```diff
@@ -365,15 +365,19 @@ The image block is wrapped in a `figure` element with the `wp-block-image` class
 
 Inputs are available to set `alt` text, `width`, and `height` attributes.
 
-Controls set either the `is-style-default` (no masking) or `is-style-circle-mask`
-class on the wrapper element. One of these classes is always present.
+Controls set either the `is-style-default` (no masking) or `is-style-circle-mask` class on the wrapper element. One of these classes is always present.
 
 Additional controls can add wrapper classes based on generated `srcset` options:
 `size-full`, `size-large`, `size-medium`, or `size-thumbnail`. Further resizing
 adds the `is-resized` class to the wrapper.
 
-Adding caption text to the image generates a `figcaption` element before the closing
-wrapper tag.
+Adding caption text to the image generates a `figcaption` element before the
+closing wrapper tag.
+
+`u-pull-inline-{amount}` class names from our [negative
+margin](/?path=/docs/utilities-spacing--negative-margin) utility classes can
+be added to the additional css section of the image block to extend the left and
+right edges of images beyond their parent prose container.
```

Comment:
I've updated this section of documentation and re-requesting review.

Reply:
I'm surprised to see the direction to use utility classes. Is it possible for us to add opinionated defaults for the breakout alignment options I see in the Gutenberg demo ("wide width," "full width")? I'd love to get these common things out of our post drafts and into our code.

<img width="621" alt="Screen Shot 2020-06-23 at 10 24 51 AM" src="https://user-images.githubusercontent.com/69633/85435096-d89bab80-b53b-11ea-97c0-0e9d099892dd.png">

Reply:
> Is it possible for us to add opinionated defaults for the breakout alignment options I see in the Gutenberg demo

It's possible! I will update this PR to account for the "wide width" and "full width" settings.

Reply:
@tylersticka I know you had a busy day, so no need to respond to this right away. 

Part of _my_ confusion was that my image block editor options looked different than your screenshot above from your Gutenberg _demo_. In my local development environment, I'm using the version of Gutenberg that ships with WordPress and my 'alignment'  options only include; left, center and right options. 

<img width="705" alt="Screen Shot 2020-08-06 at 3 36 40 PM" src="https://user-images.githubusercontent.com/1427548/89590023-d5166800-d7fb-11ea-8b42-9b5d4ad6f351.png">

As far as I can tell, this option needs to be enabled via `add_theme_support` 
https://gogutenberg.com/blocks/image/

Which appears to work...

<img width="788" alt="Screen Shot 2020-08-06 at 4 24 46 PM" src="https://user-images.githubusercontent.com/1427548/89592804-6be62300-d802-11ea-993e-4d8f07793642.png">

Unless I'm missing something, my plan is to submit a PR to the WP repo to allow for these "advanced alignment" options as well.